rand_distr/src/poisson.rs

```diff
@@ -25,17 +25,19 @@ use std::{error, fmt};
 /// use rand_distr::{Poisson, Distribution};
 ///
 /// let poi = Poisson::new(2.0).unwrap();
-/// let v: u64 = poi.sample(&mut rand::thread_rng());
+/// let v = poi.sample(&mut rand::thread_rng()) as u64;
```

Comment:
Let's not suggest this cast for now, it may result in undefined behavior for Rust < 1.45.
```suggestion
/// let v = poi.sample(&mut rand::thread_rng());
```

Reply:
I don't think it's a right place to teach users about `as`. I've simply removed the casting, so the example now uses floats.